frontend/app/controllers/spree/products_controller.rb

```diff
@@ -58,9 +77,5 @@ def redirect_if_legacy_path
         redirect_to url_for(params), status: :moved_permanently
       end
     end
-
-    def load_taxonomies
```

Comment:
@tomurb we need to keep this method in the same place

Reply:
it's above